cli/azd/pkg/input/console.go

```diff
@@ -925,13 +925,13 @@ func (c *AskerConsole) Handles() ConsoleHandles {
 }
 
 // consoleWidth the number of columns in the active console window
-func consoleWidth() int {
+func consoleWidth() int64 {
 	width, _ := consolesize.GetConsoleSize()
```

Comment:
I just threw more memory at the problem here -- there wasn't an actual overflow risk.

The underlying pixel counts are `u16`, being returned as `int`.

Happy to take suggestions.

Reply:
I ended up thinking about this more -- I don't see a future with column widths exceeding  `math.MaxInt32`. Ended up doing the conversion in this function and suppress the false-positive overflow